spec/factory_bot_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
```

Comment:
IMO you don't really need a test file for factorybot, since it'll be pretty extensively tested (albeit implicitly) through the rest of our tests.

Reply:
This is to ensure our monkey patching of the rewinding continues to work, otherwise we would get inconsistent results which may be hard to debug.